src/components/DepositWidget/useRowActions.tsx

```diff
@@ -140,11 +140,11 @@ export const useRowActions = (params: Params): Result => {
 
       const receipt = await depositApi.withdraw({ userAddress, tokenAddress, networkId, txOptionalParams })
 
-      log(`The transaction has been mined: ${receipt.transactionHash}`)
+      logDebug(`[DepositWidget:useRowActions] The transaction has been mined: ${receipt.transactionHash}`)
       toast.success(`Withdraw of ${formatAmount(pendingWithdraw.amount, decimals)} ${symbol} completed`)
     } catch (error) {
-      console.error('Error executing the withdraw request', error)
-      toast.error(`Error executing the withdraw request: ${error.message}`)
+      console.error('[DepositWidget:useRowActions] Error executing the withdraw request', error)
+      toast.error(`[DepositWidget:useRowActions] Error executing the withdraw request: ${error.message}`)
```

Comment:
Should we show that internal info in the toast notification?

Reply:
Ups! good thing you saw that :)